cmd/provider.go

```diff
@@ -27,6 +28,7 @@ func nodeTypesToTable(nodeTypes []types.NodeType) ([]ui.Column, []ui.Row) {
 			regions = strings.Join(nodeType.Regions, ", ")
 		}
 		row := ui.Row{
+			nodeType.Type,
```

Comment:
This printed table is getting pretty wide and doesn't fit on one screen. We might need to refactor it to only include minimal identifying info (name, status, type, createdAt (?), and leave the rest to the `unweave ls <exec-name>` command (which would then print the info vertically.